### PR TITLE
Bump dependencies for release v1.24.1

### PR DESCRIPTION
**Pull request by ConnorJC3**
/cc @AndrewSirenko

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ sidecars:
     image:
       pullPolicy: IfNotPresent
       repository: public.ecr.aws/eks-distro/kubernetes-csi/external-attacher
-      tag: "v4.4.0-eks-1-28-6"
+      tag: "v3.6.0-eks-1-28-7"
```
Comment (AndrewSirenko) [inline charts/aws-ebs-csi-driver/values.yaml:47]: Typo

File: charts/aws-ebs-csi-driver/values.yaml [hunk 2]
```diff
@@ -71,7 +71,7 @@ sidecars:
     image:
       pullPolicy: IfNotPresent
       repository: public.ecr.aws/eks-distro/kubernetes-csi/external-snapshotter/csi-snapshotter
-      tag: "v6.3.0-eks-1-28-6"
+      tag: "v3.6.0-eks-1-28-7"
```
Comment (AndrewSirenko) [inline charts/aws-ebs-csi-driver/values.yaml:74]: Another typo

**Comment by JacobValdemar** [comment]
Shouldn't this be featured in the CHANGELOG like it was for [v1.16.0](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/CHANGELOG.md#v1160)?

**Pull request merged**